source/module_base/math_integral.h

```diff
@@ -13,7 +13,14 @@ class Integral
     Integral();
     ~Integral();
 
-	// Peize Lin accelerate 2017-10-02
+    /**
+     * @brief simpson integral. Peize Lin accelerate 2017-10-02
```

Comment:
Author and date should be placed in a dedicated field.

Reply:
Where is the dedicated field. How about behind the function name?
![image](https://user-images.githubusercontent.com/91927439/151119554-9c846f4d-c88d-403e-b9ec-22f939e1e81c.png)

Reply:
> Where is the dedicated field. How about behind the function name? ![image](https://user-images.githubusercontent.com/91927439/151119554-9c846f4d-c88d-403e-b9ec-22f939e1e81c.png)

"author" and "date". Read more at [doxygen manual](https://www.doxygen.nl/manual/commands.html).

That's OK!

Reply:
@pxlxingliang And, I suggest you paste code in reply, but not screenshot.